molecule/shared/playbook.yml

```diff
@@ -0,0 +1,22 @@
+- name: converge
+  hosts: all
+  roles:
+    - role: passwordless_ssh
+      passwordless_ssh_public_key: "{{ role_path }}/tests/id_rsa.pub"
+      passwordless_ssh_private_key: "{{ role_path }}/tests/id_rsa"
+      passwordless_ssh_user: derp
+      passwordless_ssh_user_home: /home/derp
+  post_tasks: []
+
+- name: test, kinda
+  hosts: all[0]
+  tasks:
+    - name: try to ssh
+      command: ssh {{ hostvars[groups['all'][1]]['ansible_host'] }} ls
```

Comment:
Ansible has an [SSH module](https://docs.ansible.com/ansible/2.5/plugins/connection/ssh.html)

Reply:
That's a connection plugin, not a module. This is testing that host a and host b from the same group can connect to each other directly, which is not a use case that Ansible directly supports to my knowledge.